doc/en/parametrize.rst

```diff
@@ -79,19 +79,30 @@ them in turn:
     FAILED test_expectation.py::test_eval[6*9-42] - AssertionError: assert 54...
     ======================= 1 failed, 2 passed in 0.12s ========================
 
+.. note::
+
+    Parameter values are passed as-is to tests (no copy whatsoever).
+
+    For example, if you pass a list or a dict as a parameter value, and 
```

Comment:
Linting complains about a trailing space here, applying suggestion to remove it.

```suggestion
    For example, if you pass a list or a dict as a parameter value, and
```